File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_One/CPP/sumofnatural.cpp
/*
Program 2 : Write a Program to print Sum of First 10 Natural Numbers.
Output: The s um of First 10 Natural Numbers : 55

*/


#include<iostream>

#define lowerlimit 1
#define upperlimit 10

int main(){
	
	printf("Sum of first 10 natural number\n");
	int sum = 0;
	for(int itr = lowerlimit ; itr <= upperlimit ; itr++){
		
		sum = sum + itr ;
	}
	
	printf("%d",sum);	
}	

File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_One/JAVA/SumOfNatural.java
/*
Program 2 : Write a Program to print Sum of First 10 Natural Numbers.
Output: The s um of First 10 Natural Numbers : 55

*/

class SumOfNaturalNumber{
	
	public static void main(String args[]){
		
		System.out.printf("Sum of first 10 natural number\n");
		int sum = 0;
		for(int itr = 1 ; itr <= 10 ; itr++){

			sum = sum + itr ;
		}

		System.out.printf("%d",sum);
	}	

}	
